### ISSUE TITLE
Add Russian Translation

### ISSUE THREAD

**Issue by AYuPro** [ncase/trust#26]
url
https://ayuprogrammers.github.io/trust/
fork
https://github.com/AYuProgrammers/trust

**Comment by ncase** [comment]
Ack, someone else sent in a Russian translation yesterday! It's the one I currently have linked to from the main page: https://likzet.github.io/trust/

However, I _do_ like your translation better, since you got a nice handwritten font that works with Cyrillic! What do you think we should do? Do you want to send a PR to https://github.com/likzet/trust to include your handwritten font? Or: I could share your translation on my Twitter? I don't want your hard work to go to waste!

**Comment by AYuPro** [comment]
My translation is better. and i think no matter who send it first.
I don't need promotions and money.
If they want to take my font, let them take all the text with him, because his text very bad.
With one condition, no advertising on the main page.

**Comment by ytsygakova** [comment]
As another person who was late with Russian localization, I wouldn't say it is bad :) 
But you guys can merge it to get the best result, no doubts

**Comment by AYuPro** [comment]
I was only an hour late. And don't tell me that his machine translation is better than mine.
This man did nothing in fact, I don't need his help. But he is unable even to make the font, and puts his username on the main page, that even the author does not did.

**Comment by ytsygakova** [comment]
I agree, positioning your name on the front page is kinda immodestly (but seems like all translations now have it, so probably it was added by Nicky). And **some** phrases are translated too literally. And font, of cause :)
But your translation is not perfect either: different names for characters in the game and in the sand mode, rough phrases, line breaks. There was smth else, don't remember...
Well, imho it's better to create a forum page or sort of and let people suggest/vote the best phrases for translations.

**Comment by AYuPro** [comment]
In Russian language the endings of words usually changes a lot in text, so do not pay attention to it, the only place where is not the same, a Простак/Простачок, its for purpose, second variant less aggressive and more suitable to the description, rather than a name.

"rough phrases" - thats to convey a desired message. Otherwise its too bland..
"line breaks" - long words, I know.
